packages/core/src/accessanalyzer/vue/iamPolicyChecks.vue

```diff
@@ -20,6 +20,9 @@
                     <li>
                         <code> pip install tf-policy-validator </code>
                     </li>
+                    <li>
+                        <p>Provide IAM Roles Credentials</p>
```

Comment:
To check and make use of this, do you plan on using the built in auth system provided by the toolkit?

Reply:
Currently it only uses the shared credentials defined by the SDK. I think it will eventually use the other ways of auth in Toolkit. AFAIK, Toolkits provides a few ways to sign-in aside from IAM Roles Credentials, but if a user set their credentials using Toolkit auth system, it should pick up the same credentials since it is written to the same file.